src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -190,7 +219,11 @@ public Revision sortBranchesForRevision(Revision revision, List<BranchSpec> bran
                         }
 
                     } finally {
-                        walk.release();
+                        try {
+                            walk.release();
```

Comment:
:bug: AIUI you may not get here at all if the JVM decides to throw a LinkageError on initial processing of the bytecode. Better would be to wrap the call to `release()` into a `_release(walk)` method call and then you don't hit the issue... cf the many "dancing game" methods in Jenkins core / plugins to handle this kind of reflection dances